edgedb/con_utils.py

```diff
@@ -261,7 +260,7 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
     if user is None:
         user = os.getenv('EDGEDB_USER')
         if not user:
-            user = getpass.getuser()
+            user = 'edgedb'
```

Comment:
This was covered in another PR, right?

Reply:
Yes, I think I've mentioned that in the original PR message. Now rebased.